engines/director/detection_tables.h

```diff
@@ -408,6 +409,8 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "wallobee", 			"Wallobee Jack: The Bingi Burra Stone" },
 	{ "wallobee2", 			"Wallobee Jack: The Thai Sun Adventure" },
 	{ "wallobee3", 			"Wallobee Jack: The Lost Lionardo" },
+	{ "wallobee4",			"Wallobee Jack: Secret of the Sphinx" },
+	{ "wallobeetemple",		"The temple of Monakumba: Starring Wallobee Jack" },
```

Comment:
Should Temple be capitalized?

Reply:
The original title screen doesn't. Possibly because it's a Québecois developer and you wouldn't in French.

![scummvm-wallobeetemple-shock-00000](https://user-images.githubusercontent.com/780485/151717506-d23a3252-5dab-4b5b-a594-195e6336fc35.png)

Reply:
Ok, that makes sense